client/my-sites/plugins/plugin-navbar-header/index.jsx

```diff
@@ -0,0 +1,662 @@
+											</svg>
+											<span className="x-hidden">{ translate( 'WordPress.com' ) }</span>
+										</a>
+									</li>
+									<li className="x-nav-item x-nav-item--wide">
+										<button
+											role="menuitem"
+											className="x-nav-link x-link"
+											data-dropdown-trigger="products"
+											aria-haspopup="true"
+											aria-expanded="false"
+										>
+											{ translate( 'Products' ) }{ ' ' }
+											<span className="x-nav-link-chevron" aria-hidden="true"></span>
+										</button>
+										<div
+											className="x-dropdown-content"
+											data-dropdown-name="products"
+											role="menu"
+											aria-label={ translate( 'Products' ) }
+											aria-hidden="true"
+										>
+											<ul>
+												<li>
```

Comment:
There should be an easy way to componentize this and as @jsnajdr suggested, this would make the component way smaller and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).